pyscriptjs/src/components/pyscript.ts

```diff
@@ -139,7 +141,7 @@ export function make_PyScript(interpreter: InterpreterClient, app: PyScriptApp)
         });
 
         // bootstrap all already live py <script> tags
-        callback([{ addedNodes: document.querySelectorAll(pyScriptCSS) } as unknown] as MutationRecord[], null);
+        callback([{ addedNodes: $$(pyScriptCSS) } as unknown] as MutationRecord[], null);
```

Comment:
Similarly, maybe an explicit `document` as the root?

Reply:
done